src/api/index.ts

```diff
@@ -28,5 +28,7 @@ export const URL = {
     GET_INACTIVE_MATCHES: '/api/match/inactive',
     GET_ACTIVE_MATCHES: '/api/match/active',
     CREATE_MATCH: '/api/match',
+    RE_DRAFT: '/api/match/', //:id/draft
```

Comment:
shouldn't this be as follows?

```suggestion
    RE_DRAFT: '/api/match/:id/draft',
```

Reply:
A better solution might be,
```suggestion
    RE_DRAFT: (id: string) => '/api/match/${id}/draft',
```